layouts/partials/templates/cookie-consent.html

```diff
@@ -0,0 +1,66 @@
+<div id="cookie-notice">
+<span>We would like to use third party cookies and scripts to improve the 
+functionality of this website.</span>
+<a id="cookie-notice-accept" class="btn btn-primary btn-sm">Approve</a>
+<a id="cookie-notice-deny" class="btn btn-primary btn-sm">Deny</a>
+<script>
+    function createCookie(name,value,days) {
+        var expires = "";
+        if (days) {
+            var date = new Date();
+            date.setTime(date.getTime() + (days*24*60*60*1000));
+            expires = "; expires=" + date.toUTCString();
+        }
+        document.cookie = name + "=" + value + expires + "; path=/";
+    }
+    function readCookie(name) {
+        var nameEQ = name + "=";
+        var ca = document.cookie.split(';');
+        for(var i=0;i < ca.length;i++) {
+            var c = ca[i];
+            while (c.charAt(0)==' ') c = c.substring(1,c.length);
+            if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
+        }
+        return null;
+    }
+    function eraseCookie(name) {
+        createCookie(name,"",-1);
+    }
+
+    if(readCookie('cookie-notice-option')=='true') {
+        function loadScriptAsync(scriptSrc, callback) {
+            if (typeof callback !== 'function') {
+                throw new Error('Not a valid callback for async script load');
+            }
+            var script = document.createElement('script');
+            script.onload = callback;
+            script.src = scriptSrc;
+            document.head.appendChild(script);
+        }
+
+        /* This is the part where you call the above defined function and "calls back" 
+        your code which gets executed after the script has loaded */
+        //REPLACE WITH YOUR GA TAG!
+        loadScriptAsync('https://www.googletagmanager.com/gtag/js?id=G-W43Z7RYFEW', function () {
+            window.dataLayer = window.dataLayer || [];
+            function gtag() { dataLayer.push(arguments); }
+            gtag('js', new Date());
+            gtag('config', 'G-W43Z7RYFEW', { 'anonymize_ip': true }); //REPLACE WITH YOUR GA TAG!
```

Comment:
same here RE removing the comment

```suggestion
            gtag('config', 'G-W43Z7RYFEW', { 'anonymize_ip': true });
```